submodules/wizard/wizard.js

```diff
@@ -232,6 +232,7 @@ define(function(require) {
 					correlative: 1
 				},
 				generalSettingsData = data.generalSettings,
+				defaultCountry = _.get(monster.config, 'whitelabel.countryCode', 'US'),
```

Comment:
We usually don't want to hard code locale info in apps to avoid inconsistencies throughout the UI.

In this case, if `countryCode` is not set in `config.js`, [the framework will default to `US`](https://github.com/2600hz/monster-ui/blob/master/src/js/lib/monster.js#L50) so you can access it directly.
```suggestion
				defaultCountry = monster.config.whitelabel.countryCode,
```